prometheus.lua

```diff
@@ -475,6 +475,9 @@ local function inc_gauge(self, value, label_values)
   if err or forcible then
     self._log_error_kv(k, value, err or ERR_MSG_LRU_EVICTION)
   end
+  if self.exptime then
+    self._dict:expire(k, self.exptime)
```

Comment:
I want to know why we use `expire`? It seems that, `ttl` is another way

I do not know what is the different between them

Reply:
The `ngx.shared.DICT:ttl` is supposed to `get` the ttl of the existing key, not `set`
![image](https://github.com/api7/nginx-lua-prometheus/assets/9354193/6b7d44ed-cafb-4e59-88d7-c61bdb900c83)

Reply:
got it, thx